v3ElectronicsUserGuide.md

```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
     - Note this will kill any running process on the hardware, but if you're updating FW no one should be using the system anyway.
  31.  Summarize the actions you took in the elog entry you have already opened.
 
+## Preparing For a Power Cut
+
+In order to safely prepare a GEM test stand before a planned power cut execute:
+
+ 1. Power down the High Voltage.
+     - Use the opportunity to power down other sensitive equipement such as PMT's.
+ 2. Power down the Low Voltage.
+ 3. Place the µTCA modules in extraction mode:
+     - Gently pull the hot swap tab on the AMC's (CTP7 and AMC13). Wait until the blue LED stays on solid on each AMC's.
```

Comment:
Change " on the AMC's (CTP7 and AMC13)" to "on all AMC's including the AMC13" as some users might be using GLIBs.